Source/WebKit/WebProcess/WebProcess.h

```diff
@@ -501,6 +501,7 @@ class WebProcess final : public AuxiliaryProcess {
 
 #if PLATFORM(COCOA)
     void registerAdditionalFonts(AdditionalFonts&&);
+    void registerFontMap(HashMap<String, URL>&&, Vector<SandboxExtension::Handle>&& sandboxExtensions);
```

Comment:
Should we remove `registerAdditionalFonts` now that it's replaced by `registerFontMap`?